Framework/Algorithms/test/PDCalibrationTest.h

```diff
@@ -475,6 +477,104 @@ class PDCalibrationTest : public CxxTest::TestSuite {
+    TS_ASSERT_THROWS_NOTHING(wsalg.initialize());
+    TS_ASSERT(wsalg.isInitialized());
+    TS_ASSERT_THROWS_NOTHING(wsalg.setPropertyValue("OutputWorkspace", "ws"));
+    TS_ASSERT_THROWS_NOTHING(wsalg.setPropertyValue("WorkspaceType", "Event"));
+    TS_ASSERT_THROWS_NOTHING(wsalg.setPropertyValue("Function", "User Defined"));
+    TS_ASSERT_THROWS_NOTHING(wsalg.setPropertyValue("UserDefinedFunction", function.str()));
+    TS_ASSERT_THROWS_NOTHING(wsalg.setProperty("XMin", 1.0));
+    TS_ASSERT_THROWS_NOTHING(wsalg.setProperty("XMax", 16666.7));
+    TS_ASSERT_THROWS_NOTHING(wsalg.setProperty("BinWidth", 1.0));
+    TS_ASSERT_THROWS_NOTHING(wsalg.setProperty("NumEvents", 100000));
+    TS_ASSERT_THROWS_NOTHING(wsalg.setProperty("BankPixelWidth", 1));
+    TS_ASSERT_THROWS_NOTHING(wsalg.setProperty("NumBanks", 1));
+    TS_ASSERT_THROWS_NOTHING(wsalg.execute());
+    TS_ASSERT(wsalg.isExecuted());
+
+    const std::string params = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+                               "<parameter-file instrument=\"basic_rect\" valid-from=\"1900-01-31T23:59:59\">\n"
+                               "<component-link name=\"basic_rect\">\n"
+                               "<parameter name=\"IkedaCarpenterPV:Alpha0\" type=\"fitting\" >\n"
+                               "       <formula eq=\"1.6\" result-unit=\"TOF\"/>\n"
+                               "        <fixed />\n"
+                               "</parameter>\n"
+                               "<parameter name=\"IkedaCarpenterPV:Alpha1\" type=\"fitting\" >\n"
+                               "        <formula eq=\"1.5\" result-unit=\"TOF\"/>\n"
+                               "        <fixed />\n"
+                               "</parameter>\n"
+                               "<parameter name=\"IkedaCarpenterPV:Beta0\" type=\"fitting\" >\n"
+                               "        <formula eq=\"31.7927\" result-unit=\"TOF\"/>\n"
+                               "        <fixed />\n"
+                               "</parameter>\n"
+                               "<parameter name=\"IkedaCarpenterPV:Kappa\" type=\"fitting\" >\n"
+                               "        <formula eq=\"0.1\"/>\n"
+                               "        <fixed />\n"
+                               "</parameter>\n"
+                               "    <parameter name=\"IkedaCarpenterPV:SigmaSquared\" type=\"fitting\">\n"
+                               "        <formula eq=\"1.0\" unit=\"dSpacing\" result-unit=\"TOF^2\"/>\n"
+                               "    </parameter>\n"
+                               "    <parameter name=\"IkedaCarpenterPV:Gamma\" type=\"fitting\">\n"
+                               "        <formula eq=\"1.0\" unit=\"dSpacing\" result-unit=\"TOF\"/>\n"
+                               "    </parameter>\n"
+                               "</component-link>\n"
+                               "</parameter-file>";
+
+    LoadParameterFile paramalg;
```

Comment:
Setting this shouldn't be required as these should all be the default values anyway.